Source/JavaScriptCore/bytecode/PropertyCondition.cpp

```diff
@@ -337,6 +379,7 @@ bool PropertyCondition::isStillValid(Concurrency concurrency, Structure* structu
     // absence. It doesn't affect AbsenceOfSetEffect because impure properties aren't ever setters.
     switch (m_header.type()) {
     case Absence:
+    case AbsenceOfIndexed:
```

Comment:
is this needed for indexed accesses?

Reply:
If this is just you being conservative, maybe just say this might be more conservative than we need to be

Reply:
Yeah, right now it is added conservatively. I'll drop it.